account_reconcile_oca/models/account_bank_statement_line.py

```diff
@@ -717,3 +717,17 @@ def action_to_check(self):
     def action_checked(self):
         self.ensure_one()
         self.move_id.to_check = False
+
+    def _get_reconcile_line(
+        self, line, kind, is_counterpart=False, max_amount=False, from_unreconcile=False
+    ):
+        vals = super()._get_reconcile_line(
+            line,
+            kind,
+            is_counterpart=is_counterpart,
+            max_amount=max_amount,
+            from_unreconcile=from_unreconcile,
+        )
+        if vals["partner_id"] is False:
+            vals["partner_id"] = (False, self.partner_name)
```

Comment:
Is this reconcile line a dictionary for feeding the widget and not one for creating a record? If so, it's better to add `vals["partner_name"]` and show it in the UI if the other value doesn't exist.